src/components/Footer.test.js

```diff
@@ -0,0 +1,14 @@
+import { screen, render } from '@testing-library/react';
+import React from 'react'
+import Footer from './Footer'
+
+
+describe('Footer', () => {
+    it('Displays my copyright message', () => {
+        render(<Footer/>);
+
+        expect(screen.getByText("2023 by Alison Childers", {
```

Comment:
Would reccomend mocking, or using something called dependency injection/inversion of control to pass either the date INTO the component, or simply mock the Date function that you've used inside the component. Can definitely explain these concepts with you on a call if you like! 


Here is a link so you can read up on your own:
 https://softwareengineering.stackexchange.com/questions/140992/is-dependency-injection-essential-for-unit-testing

Reply:
Ah ok yes this makes sense! Thanks!